### PR TITLE
feat: Added seaskipper locations for Artemis

### PR DESCRIPTION
**Pull request by JamieCallan117**
Needs md5, unsure how to find that

### CODE REVIEW

**Comment by kristofbolyai** [comment]
You can search for "how to get md5sum for file in <os>"

**Comment by kristofbolyai** [comment]
> And I just do that for the file in my local repository? Doesn't have to be downloaded from the main repository?

The MD5 hash itself does not change, as it is linked to that file with that specific content.

File: seaskipper_locations.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    },
+    {
+        "destination": "Zhight Island",
+        "combatLevel": 55,
+        "startX": -727,
+        "startZ": -2990,
+        "endX": -440,
+        "endZ": -2629
+    }
+]
```
Comment (kristofbolyai) [inline seaskipper_locations.json:154]: Please put a newline here :)

**Review by magicus — APPROVED** [review]

**Comment by magicus** [comment]
I added this but under a different name.

**Pull request closed without merging**